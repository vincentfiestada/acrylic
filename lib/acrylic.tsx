/**
 * Re-export all components in one place
 */
export { default as IProps } from "./components/IProps";
export { default as IEventProps } from "./components/IEventProps";

/**
 * Layout Components
 */
export { default as ILayoutProps } from "./components/layout/ILayoutProps";
export { default as Section } from "./components/layout/Section";
export { default as Column } from "./components/layout/Column";
export { default as Row } from "./components/layout/Row";
export { default as Alert } from "./components/layout/Alert";
export { default as Toolbar } from "./components/layout/Toolbar";

/**
 * Typography Components
 */
export { default as ITextProps } from "./components/typography/ITextProps";
export { default as Title } from "./components/typography/Title";
export { default as Text } from "./components/typography/Text";
export { default as Icon } from "./components/typography/Icon";

/**
 * Control Components
 */
export { default as Button } from "./components/controls/Button";
export { default as IconButton } from "./components/controls/IconButton";

/**
 * Variants
 */
export { default as Variant } from "./components/Variant";
