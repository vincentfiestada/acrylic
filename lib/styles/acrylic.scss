@import "normalize.css";
@import "typography";
@import "controls";
