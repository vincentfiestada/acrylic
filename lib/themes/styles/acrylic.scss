@import "normalize.css";
@import "typography";
@import "icons";
@import "layout";
@import "controls";
@import "progress";
@import "animation";
