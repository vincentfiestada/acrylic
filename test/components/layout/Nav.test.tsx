import * as React from "react";
import { mount } from "enzyme";
import { Nav, Menu, MenuItem } from "../../../lib/acrylic";

describe("<Nav />", () => {
	it("Should match the latest snapshots", () => {
		expect(
			mount(
				<Nav>
					<Menu>
						<MenuItem>Item</MenuItem>
						<MenuItem>Item</MenuItem>
						<MenuItem>Item</MenuItem>
					</Menu>
				</Nav>
			)
		).toMatchSnapshot();
	});
});
