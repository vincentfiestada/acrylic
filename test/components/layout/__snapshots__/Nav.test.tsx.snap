// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> Should match the latest snapshots 1`] = `
<Nav>
  <nav
    className="acr-section acr-nav"
  >
    <Menu>
      <ul
        className="acr-menu"
      >
        <MenuItem>
          <li
            className="acr-menu-item acr-li-text-only"
          >
            Item
          </li>
        </MenuItem>
        <MenuItem
          active={true}
        >
          <li
            className="acr-menu-item active acr-li-text-only"
          >
            Item
          </li>
        </MenuItem>
        <MenuItem>
          <li
            className="acr-menu-item acr-li-text-only"
          >
            Item
          </li>
        </MenuItem>
      </ul>
    </Menu>
  </nav>
</Nav>
`;
